doc(backup): multiple backup stores support (#1035)
longhorn/website: content/docs/1.8.0/advanced-resources/system-backup-restore/backup-longhorn-system.md

- StorageClasses
- Volumes

> **Note:**
>
> - The default backup target (`default`) is always used to store system backups.
> - Longhorn is unable to back up V2 Data Engine backing images.
> - Longhorn does not backup `Nodes`. The Longhorn manager on the target cluster is responsible for creating its own Longhorn `Node` custom resources.

> > - Longhorn system backup bundle only includes resources operated by Longhorn.  

Author: Modified. Thanks

- StorageClasses
- Volumes

> > **Note:** System backups are stored exclusively via the default backup target `default`.

Reviewer: ```suggestion
> **Note:** The default backup target (`default`) is always used to store system backups.
```

Author: Modified. Thanks

> **Note:** Longhorn is unable to back up V2 Data Engine backing images.

> > **Note:** Longhorn does not backup `Nodes`. The Longhorn manager on the target cluster is responsible for creating its own Longhorn `Node` custom resources.

Reviewer 2: Too many `Note`. Please use one not for the four items

![image](https://github.com/user-attachments/assets/a962b451-ad91-4d0b-b6cd-6f2171839105)

> Note
>
> - ...
>
> - ....

Author: Modified. Thanks.
